.github/workflows/python-tests.yml

```
@@ -9,17 +9,18 @@ jobs:
   test:
     runs-on: ubuntu-latest
     env:
-      RPC_URL: ${{ secrets.RPC_URL }}
-      BACKUP_WALLET: ${{ secrets.BACKUP_WALLET }}
+      GNOSIS_RPC_URL: ${{ secrets.GNOSIS_RPC_URL }}
+      MODIUS_RPC_URL: ${{ secrets.MODIUS_RPC_URL }}
```

```suggestion
      MODE_RPC_URL: ${{ secrets.MODE_RPC_URL }}
```
the chain is called Mode

---

@Mohsinsiddi pls consider this in the PR for Memeooorr's test